packages/fractor-typoscript/src/TypoScriptFileProcessor.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\FractorTypoScript;
+
+use a9f\Fractor\Application\Contract\FileProcessor;
+use a9f\Fractor\Application\ValueObject\File;
+use a9f\FractorTypoScript\Contract\TypoScriptFractor;
+use Helmich\TypoScriptParser\Parser\Parser;
+use Helmich\TypoScriptParser\Parser\Printer\PrettyPrinter;
+use Helmich\TypoScriptParser\Parser\Printer\PrettyPrinterConfiguration;
+use Symfony\Component\Console\Output\BufferedOutput;
+
+/**
+ * @implements FileProcessor<TypoScriptFractor>
+ */
+final readonly class TypoScriptFileProcessor implements FileProcessor
+{
+    /**
+     * @param iterable<TypoScriptFractor> $rules
+     */
+    public function __construct(
+        private iterable $rules,
+        private Parser $parser,
+        private PrettyPrinter $printer
+    ) {
+    }
+
+    public function canHandle(File $file): bool
+    {
+        return in_array($file->getFileExtension(), ['typoscript', 'ts']);
```

Comment:
What about user and page ts config? I think they have their own file extension sometimes

Reply:
Yeah, and there is ext_typoscript_setup.txt as well. I'll add tsconfig for now and a comment that this should be configurable in the future.